### PR TITLE
🚀 Automate Connections with GitHub Actions! 

### LINKED ISSUE #336: 🚀 Automate Connections with GitHub Actions!
We can implement github workflows for the following things:

Auto comment on issue Closed and Opened
Auto comment on PR mergs
Auto comment on PR raises
Using this workflows makes you'r repo more interactive !

please assign me and i'm expecting a level3 with gssoc-ext and hacktoberfest-accepted label
@Vignesh025

### PR DESCRIPTION
**Pull request by amiya-cyber**
## 📋 Description

Auto comment on issue Closed and Opened
Auto comment on PR mergs
Auto comment on PR raises

## 🔨 Changes Made

Fixes #336 

- made changes in workflow
- 
- 

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [x ] My code follows the style guidelines of this project.
- [ x] I have performed a self-review of my own code.
- [ x] I have commented my code, particularly in hard-to-understand areas.
- [ x] I have made corresponding changes to the documentation (if applicable).
- [ x] My changes generate no new warnings.
- [ x] I have tested it locally and it works fine.
- [ x] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [ ] Bug fix (non-breaking change which fixes an issue) 🐛
- [ x] New feature (non-breaking change which adds functionality) ✨
- [ ] UI enhancement (non-breaking change which enhances UI) 🎨
- [ ] Documentation update 📚

## 🤝 Related Issues

- Fixes #

## 📸 Screenshots (if applicable)

---

## Reminder:
Please ensure your pull request is made to the `test` branch, not `main`.

*Thank you for contributing to VigneshDevHub! We look forward to reviewing your PR and getting it merged!*

please add level2 or level3 label @Vignesh025

### CODE REVIEW

**Comment by amiya-cyber** [comment]
now i think this code will work file please review the pr and add label
thank you @Vignesh025

**Comment by Vignesh025** [comment]
@amiya-cyber It seems like this code has been copied from another project. Please work on creating an original solution, or I’ll have to close this PR.

**Comment by amiya-cyber** [comment]
No this code is not copy pasted and this workflow works nicely I hope you get my point thank you @Vignesh025 please review and merge

**Comment by Vignesh025** [comment]
Okay, adjust it according to our project.

**Comment by amiya-cyber** [comment]
I think this will work nicely if you can merge it thank you @Vignesh025

File: .github/workflows/pr-raise.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  comment:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+    - name: Add Comment to Pull Request
+      run: |
+        COMMENT=$(cat <<EOF
+        {
+          "body": "Thank you for submitting your pull request! 🙌 We'll review it as soon as possible. In the meantime, please ensure that your changes align with our chaotic [CONTRIBUTING.md](https://github.com/vansh-codes/ChaosWeb/blob/main/CONTRIBUTING). If there are any specific instructions or feedback regarding your PR, we'll provide them here. Thanks again for your contribution! 😊"
```
Comment (Vignesh025) [inline .github/workflows/pr-raise.yml:23]: change this according to our project. Only then I can merge it.

Comment (Vignesh025) [inline .github/workflows/pr-raise.yml:23]: Change this according to our project, only then I can merge it.

**Pull request closed without merging**